langchain-core/src/prompts/serde.ts

```diff
@@ -13,7 +14,7 @@ export type SerializedPromptTemplate = {
   _type?: "prompt";
   input_variables: string[];
   template_format?: TemplateFormat;
-  template?: string;
+  template?: MessageContent;
```

Comment:
Is this necessary?

If it's breaking, we may as well deprecate the whole thing

Reply:
Yes to necessary, but I don't believe this would be breaking since `template` can be `string`

Or I'm missing context